lib/compilers/pythran.ts

```diff
@@ -0,0 +1,42 @@
+import path from 'path';
+import {BaseCompiler} from '../base-compiler.js';
+import type {PreliminaryCompilerInfo} from '../../types/compiler.interfaces.js';
+import type {ParseFiltersAndOutputOptions} from '../../types/features/filters.interfaces.js';
+
+export class PythranCompiler extends BaseCompiler {
+    static get key() {
+        return 'pythran';
+    }
+
+    cpp_compiler_root: string;
+
+    constructor(info: PreliminaryCompilerInfo, env) {
+        super(info, env);
+        this.cpp_compiler_root = this.compilerProps<string>(`compiler.${this.compiler.id}.cpp_compiler_root`);
+    }
+
+    override getDefaultExecOptions() {
+        const execOptions = super.getDefaultExecOptions();
+        if (this.cpp_compiler_root) {
+            execOptions.env.PATH = path.join(this.cpp_compiler_root, 'bin');
+            execOptions.env.LD_LIBRARY_PATH = path.join(this.cpp_compiler_root, 'lib');
```

Comment:
I think for completeness you also need `lib64`

Reply:
oh right. I think I had it at some point, but I must have dropped this part by mistake. Thanks!